tools/admin/src/main/scala/whisk/core/cli/Main.scala

```diff
@@ -0,0 +1,163 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package whisk.core.cli
+
+import java.io.File
+
+import akka.actor.ActorSystem
+import akka.stream.ActorMaterializer
+import ch.qos.logback.classic.{Level, LoggerContext}
+import org.rogach.scallop._
+import org.slf4j.LoggerFactory
+import pureconfig.error.ConfigReaderException
+import whisk.common.{AkkaLogging, Logging, TransactionId}
+import whisk.core.database.{LimitsCommand, UserCommand}
+
+import scala.concurrent.duration.DurationInt
+import scala.concurrent.{Await, Future}
+import scala.util.{Failure, Success, Try}
+
+class Conf(arguments: Seq[String]) extends ScallopConf(arguments) {
+  banner("OpenWhisk admin command line tool")
+
+  val verbose = tally()
+  val configFile = opt[File](descr = "application.conf path")
```

Comment:
as a future enhancement, should we permit environment based overrides or using the command line to specify a configuration inline?

Reply:
That should work as of now also as the config loading is done in the standard way like being done when Controller/Invoker run in container